.azure-pipelines/pipelines.yml

```diff
@@ -220,6 +253,40 @@ stages:
         rbe: false
         artifactSuffix: ".arm64"
         bazelBuildExtraOptions: "--sandbox_base=/tmp/sandbox_base"
+  - job: package
+    dependsOn: []
+    timeoutInMinutes: 120
+    pool: "arm-large"
+    steps:
+    - template: bazel.yml
+      parameters:
+        managedAgent: false
+        ciTarget: bazel.packaging
+        rbe: false
+        artifactSuffix: ".arm64"
+        bazelBuildExtraOptions: "--sandbox_base=/tmp/sandbox_base"
+  - job: verify
+    # possibly move this job to verify at the end.
```

Comment:
as per the comment this could potentially run in parallel with the verifying of examples at the end